cmd/kops/create_cluster.go

```diff
@@ -174,7 +174,7 @@ func (o *CreateClusterOptions) InitDefaults() {
 	o.Yes = false
 	o.Target = cloudup.TargetDirect
 	o.Models = strings.Join(cloudup.CloudupModels, ",")
-	o.Networking = "kubenet"
+	o.Networking = "calico"
```

Comment:
This seems to be an unrelated change.

Reply:
Yes. Pretty much the only way to test if this fixes the failing e2e test.